File: androidx/src/androidTest/java/me/panpf/androidx/test/TestSupportActivity.java
package me.panpf.androidx.test;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v4.app.Fragment;
import android.support.v4.app.FragmentActivity;

public class TestSupportActivity extends FragmentActivity {

    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        this.setContentView(R.layout.at_test);
        this.getSupportFragmentManager().beginTransaction().replace(R.id.testAt_frame, new Fragment()).commit();
    }
}

File: androidx-kt/src/androidTest/java/me/panpf/androidxkt/test/TestSupportActivity.kt
package me.panpf.androidxkt.test

import android.os.Bundle
import android.support.v4.app.Fragment
import android.support.v4.app.FragmentActivity

class TestSupportActivity : FragmentActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        setContentView(me.panpf.androidxkt.test.R.layout.at_test)

        supportFragmentManager.beginTransaction().replace(me.panpf.androidxkt.test.R.id.testAt_frame, Fragment()).commit()
    }
}